docs/en/start/trace/skywalking.md

```diff
@@ -0,0 +1,60 @@
+# Skywalking trace 
+
+## Configuration
+
+Example: configs/config_trace_skywalking.json
+
+````json
+{
+  "tracing": {
+    "enable": true,
+    "driver": "SkyWalking",
+    "config": {
+      "reporter": "gRPC",
+      "backend_service": "127.0.0.1:11800",
+      "service_name": "layotto"
+    }
+  }
+}
+````
+
+| Field            | Required fields | Description  |
+|------------------|-----|--------------------------|
+| reporter         | Y   | Reporting method grpc               |
+| backend_service  | Y   | skywalking oap server address |
+| service_name     | Y   | Service Name                     |
+
+## Run skywalking
+
+````shell
+cd diagnostics/skywalking
+
+docker-compose -f skywalking-docker-compose.yaml up -d
+````
```

Comment:
Could u make this doc "runnable" too ?
You can take the chinese version as an example, see https://github.com/mosn/layotto/blob/main/docs/zh/start/trace/skywalking.md?plain=1